CHANGES_NEXT_RELEASE

```diff
@@ -32,5 +32,5 @@ Add:  Hardening: mongoBackend checking for field existence and proper type to av
 Add:  githash in --version (Issue #1363)
 Add:  Added CLI argument for notification mode. Default mode is 'transient' (previous version used 'permanent' implicitly) (Issue #1370)
 Add:  Added CLI argument for connection memory limit.  Default value is 64Kb (Issue #1384)
-Fix:  Separate semahpore for subscription cache, so that sub cache is independent of mutex policy (Issue #1401)
```

Comment:
This is already "integrated" in another line. Maybe it reapeared again due to some merge/rebase conflict resolution...

NTC